atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/domain/models/keyManagement/ExportableImportableKey.kt

```diff
@@ -0,0 +1,77 @@
+package io.iohk.atala.prism.walletsdk.domain.models.keyManagement
+
+import io.iohk.atala.prism.apollo.base64.base64UrlDecoded
+import io.iohk.atala.prism.apollo.base64.base64UrlDecodedBytes
+import io.iohk.atala.prism.apollo.base64.base64UrlEncoded
+import kotlinx.serialization.Serializable
+
+interface ExportableKey {
+    fun getPem(): String
+    fun getJwk(): JWK
+
+    fun jwkWithKid(kid: String): JWK
+}
+
+interface ImportableKey {
+    @Throws(Exception::class)
+    fun initializeFromPem(pem: String)
+
+    @Throws(Exception::class)
+    fun initializeFromJwk(jwk: JWK)
+}
+
+@Serializable
+data class JWK(
+    val kty: String,
+    val alg: String? = null,
+    val kid: String? = null,
+    val use: String? = null,
+    val n: String? = null,
+    val e: String? = null,
+    val d: String? = null,
+    val p: String? = null,
+    val q: String? = null,
+    val dp: String? = null,
+    val dq: String? = null,
+    val qi: String? = null,
+    val crv: String? = null,
+    val x: String? = null,
+    val y: String? = null,
+    val k: String? = null
+)
+
+data class PEMKey(val keyType: String, val keyData: ByteArray) {
+    constructor(keyType: String, keyData: String) : this(keyType, keyData.base64UrlDecodedBytes)
+
+    fun pemEncoded(): String {
+        val base64Data = keyData.base64UrlEncoded
```

Comment:
It should be base64 padded and not the URL version as per the RFC https://www.rfc-editor.org/rfc/rfc7468
Also, the base64 string value should be divided into lines with a max of 64 char per line

Reply:
Done